### PR TITLE
changes done

### PR DESCRIPTION
**Pull request by ruckeran**
- excluded knitr
- changed sav_list in sav_path
- adapted numbers of data set names
- changed order of "changeValLabels"

### CODE REVIEW

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 1]
```diff
@@ -10,18 +10,20 @@ vignette: >
 
 In the following, the FDZ data preparation is documented.
 
-```{r setup}
+```{r setup, include = FALSE}
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:13]: LGTM!

File: vignettes/f [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"oldValue","newValue"
```
Comment (beckerbenj) [inline vignettes/f:1]: This temporary file is created as a side effect of the vignette. We should either a) clean this up, via a not displayed code chunk that unlinks `f` or b) set the `IDrecode` chunck to `eval = FALSE`. I would suggest b), but both are OK with me.

**Review by beckerbenj — CHANGES_REQUESTED** [review]
Most looks good to me, only some minor nit and a technical issue with the temp file :)

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 3]
```diff
@@ -44,7 +44,7 @@ dat1 <- checkMissingsByValues(dat1, missingValues = -50:-99)
 # -----------------------------------------------------------------------------
 ## Recode IDs
 The first step is to create the path for the template.
-```{r IDrecode}
+```{r IDrecode, eval = FALSE}
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:47]: 👍  but this triggers problems in later code chunks. Let's discuss this after lunch.

**Review by beckerbenj — CHANGES_REQUESTED** [review]

File: vignettes/FDZ_Data_Cleaning.Rmd (hunk 3 above)
Comment (ruckeran) [reply vignettes/FDZ_Data_Cleaning.Rmd:46]: true, next time I'll knit it first!

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 4]
```diff
@@ -8,75 +8,93 @@ vignette: >
...
 
-#------------------------------------------------------------------------------
-#### example data set #####
-# loading a single data set (make sure to select the right one)
+# -----------------------------------------------------------------------------
+Loading a single data set (be careful to select the right one)
 ```{r load}
-dat <- import_spss(sav_list[1])
+dat <- import_spss(sav_path)
+str(dat$dat) # for a small overview in the variables of the data set
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:28]: 👍

**Review by beckerbenj — APPROVED** [review]
Looks all good to me! 
My suggestion: Delete file `f` and merge the PR 👍

**Pull request merged**